### DISCUSSION TITLE
Are plots independent?

### DISCUSSION THREAD

**Discussion by YahorL** [Chia-Network/chia-blockchain#3964] [Q&A]
I don't understand how plots are working.
First of all, are the independent from my wallet? Can I, for example, give my plots to a friend and he will farm on them? I made new wallet on my computer and found out, that my plots synced with new wallet and started farming.  Is this mean, that I can copy my plots and we will mine parallel and the reward for block will double and each of us will recieve 2 coins?
Secondly, I don't understand how pool is working. I've already read this article https://www.chia.net/2020/11/10/pools-in-chia.html but hpool is a bit illegal and it is not working as chia pools should. My plots, that I've mined via original chia app are not working with hpool (I don't recieve any reward, but in profile the status is Ok and I can see my net space). As I understood, I just give my plots to the pool and the it give me my reward, so why should I replot my plots.

**Comment by moshenskyDV** [comment]
No, you cannot transfer your plots through few accounts. Plot has something like "digital fingerprint".
But actually you can create plots especially for friend with `-a` option (provide friend key.

```chia plots create -h
Usage: chia plots create [OPTIONS]

Options:
-a, --alt_fingerprint INTEGER Enter the alternative fingerprint of the key you want to use
-f, --farmer_public_key TEXT Hex farmer public key
-p, --pool_public_key TEXT Hex public key of pool```

**Reply by YahorL** [reply]
But if you make another wallet on your machine, plots will automatically start farming on your new wallet. If the plots are linked to your account, how that's possible?

**Reply by moshenskyDV** [reply]
I haven't tried, but I suppose if this plots find the solution under second wallet it will be written to blockchain for the first wallet.

**Reply by YahorL** [reply]
So can I somewhere find this digital fingerprint of my plot and check if it is real my plot?

**Reply by moshenskyDV** [reply]
I'll make video about public keys stuff explanation tomorrow. Please be in touch, I'll publish it here.

**Reply by moshenskyDV** [reply]
There is answer on your questions. Cheers!
https://www.youtube.com/watch?v=dYFYawhI5i0

**Reply by N0rdmann** [reply]
So -a option only works when i got multiple CAs on my pc and want to choose for which to create a plot for without providing farmer/public key right?

**Reply by moshenskyDV** [reply]
@N0rdmann yep, exactly
